src/layouts/SidebarMenu.tsx

```diff
@@ -264,6 +273,73 @@ const SubscribedEthBalance = withSubscription({
   },
 });
 
+/***** DAO Debt *****/
+interface IDebtProps extends ISubscriptionProps<any> {
+  dao: IDAOState;
+}
+
+const DaoDebt = (props: IDebtProps) => {
+  const { data, error, isLoading } = props;
+
+  if (isLoading || error || (data === null)) {
+    return null;
+  }
+
+  const result = getDaoDebt(data);
+
+  return (
+    <>
+      {Object.keys(result).map((sym) => {
+        return (
+          <li key={"Debt_" + sym}> 
+            <strong> - {formatTokens(result[sym])} </strong> {sym}
+          </li>
+        );
+      })}
+    </>
+  );
+};
+
+const SubscribedDaoDebt = withSubscription({
+  wrappedComponent: DaoDebt,
+  checkForUpdate: (oldProps: IDebtProps, newProps: IDebtProps) => {
+    return oldProps.dao.address !== newProps.dao.address;
+  },
+  createObservable: async (props: IDebtProps) => {
+
+    const arc = getArc();
+    const query = gql`query
+    {
+      proposals(
+        where: {
+          dao: "${props.dao.address}",
+          accountsWithUnclaimedRewards_not: []
```

Comment:
this is, potentially, an ENORMOUS query, as this includes also all proposals in which some user has not claimed his rep yet. If we cannot do this with a  reasonably sized query, we will need to update the subgraph to keep the number for us.